### PR TITLE
Add initial structure and module migration

### PR DESCRIPTION
**Pull request by rainerleber**
##### SUMMARY
Add/Change initial strucutre and migrate sap modules from community.general.

##### ISSUE TYPE
-

##### COMPONENT NAME
.

##### ADDITIONAL INFORMATION
-

### CODE REVIEW

**Comment by Andersson007** [comment]
@rainerleber hi, thanks for working on this!
You can be interested in the maintainer docs that live in this [repo](https://github.com/ansible/community-docs/), in particular:
- [Maintainer guidelines](https://github.com/ansible/community-docs/blob/main/maintaining.rst)
- [Releasing guidelines](https://github.com/ansible/community-docs/blob/main/releasing_collections.rst)
- [Review checklist](https://github.com/ansible/community-docs/blob/main/review_checklist.rst) and the others

If once you wanna see the collection included in Ansible package, I would highly recommend carefully reading [Collection requirements](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst) (there's also a [short checklist](https://github.com/ansible-collections/overview/blob/main/collection_checklist.md) for convenience). It's better and easier to make the collection's stuff satisfying them now, at the beginning, than later.

**Comment by rainerleber** [comment]
@Andersson007 thank you for your input :-) . Hopefully i have prepared all, to include this collection in the Ansible package.
So what are the next steps? After i release a version is this automatically integrated in the galaxy?

**Comment by rainerleber** [comment]
@Andersson007 could you also add "rkpobe" to this repo?

**Comment by Andersson007** [comment]
> @Andersson007 thank you for your input :-) . Hopefully i have prepared all, to include this collection in the Ansible package. So what are the next steps? After i release a version is this automatically integrated in the galaxy?

@rainerleber you need to submit the collection for inclusion, see https://github.com/ansible-collections/ansible-inclusion for details. Thank you

**Comment by Andersson007** [comment]
> @Andersson007 could you also add "rkpobe" to this repo?

Invited them with "maintainer" permissions

BTW @tbludau 's invitation is still pending. Is he still interested in maintaining this collection or we can remove him?

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+notifications: true
```
Comment (Andersson007) [inline .github/BOTMETA.yml]:
```suggestion
notifications: true
```
is this file needed? Maintainers could just subscribe to this repo. If it's needed, the bot should be configured for the repo

Comment (Andersson007) [reply .github/BOTMETA.yml]: i guess the ignore.txt files can also be removed

Comment (rainerleber) [reply .github/BOTMETA.yml]: > > @Andersson007 could you also add "rkpobe" to this repo?
> 
> Invited them with "maintainer" permissions
> 
> BTW @tbludau 's invitation is still pending. Is he still interested in maintaining this collection or we can remove him?
Hi @Andersson007,
i think tbludau is not interested any more. So yes you can remove him.

Comment (rainerleber) [reply .github/BOTMETA.yml]: Thank you for the input i removed the files :-)

Comment (Andersson007) [reply .github/BOTMETA.yml]: You're welcome! FYI I cancelled the invite
